k8s/nginx/nginx-dep.yaml

```diff
@@ -20,22 +20,24 @@ spec:
       containers:
       - name: nginx
         image: bigchaindb/nginx:1.0
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
         env:
         - name: MONGODB_FRONTEND_PORT
-          value: $(NGX_INSTANCE_0_SERVICE_PORT_NGX_PUBLIC_MDB_PORT)
+          value: "27017"
         - name: MONGODB_BACKEND_HOST
-          # NGINX requires FQDN to resolve names
           value: mdb-instance-0.default.svc.cluster.local
         - name: MONGODB_BACKEND_PORT
           value: "27017"
         - name: BIGCHAINDB_FRONTEND_PORT
-          value: $(NGX_INSTANCE_0_SERVICE_PORT_NGX_PUBLIC_BDB_PORT)
+          value: "80"
         - name: BIGCHAINDB_BACKEND_HOST
-          # NGINX requires FQDN to resolve names
           value: bdb-instance-0.default.svc.cluster.local
         - name: BIGCHAINDB_BACKEND_PORT
           value: "9984"
+        - name: BIGCHAINDB_WS_FRONTEND_PORT
+          value: "81"
+        - name: BIGCHAINDB_WS_BACKEND_PORT
+          value: "9985"
```

Comment:
The following block is repeated twice, once here and once below (in the same `nginx-dep.yaml` file).
```text
         - name: BIGCHAINDB_WS_FRONTEND_PORT
           value: "81"
         - name: BIGCHAINDB_WS_BACKEND_PORT
           value: "9985"
```